wiki/Tournaments/GTS/KGTS_2019/en.md

```diff
@@ -0,0 +1,284 @@
+| **Ikkun** ![][flag_IT] | **7** | 4 | ![][flag_ES] Maitgon | [#1](https://osu.ppy.sh/community/matches/55537726) |
+| Maitgon ![][flag_ES] | 0 | **7** | ![][flag_CA] **janitoreihil** | [#1](https://osu.ppy.sh/community/matches/55541019) |
+| Ikkun ![][flag_IT] | 1 | **7** | ![][flag_CA] **janitoreihil** | [#1](https://osu.ppy.sh/community/matches/55545515) |
+| janitoreihil ![][flag_CA] | 5 | **7** | ![][flag_IT] **Ikkun** | [#1](https://osu.ppy.sh/community/matches/55547569) |
+
+### Week 1
+
+| Saturday, 2019-10-05 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-- |
+| **janitoreihil** ![][flag_CA] | **4** | 0 | ![][flag_JP] Saikoro | [#1](https://osu.ppy.sh/community/matches/55327557) |
+| **Ikkun** ![][flag_IT] | **4** | 0 | ![][flag_MY] imnoobatosuosu | [#1](https://osu.ppy.sh/community/matches/55334605) |
+| **POCARI-SWEAT** ![][flag_KR] | **4** | 2 | ![][flag_GB] Metzo | [#1](https://osu.ppy.sh/community/matches/55335417) |
+| **Maitgon** ![][flag_ES] | **4** | 0 | ![][flag_AU] Beat43210 | -win by default- |
+| imnoobatosuosu ![][flag_MY] | 2 | **4** | ![][flag_GB] **Metzo** | [#1](https://osu.ppy.sh/community/matches/55336681) |
+| **Ikkun** ![][flag_IT] | **4** | 0 | ![][flag_KR] POCARI-SWEAT | [#1](https://osu.ppy.sh/community/matches/55338000) |
+| **seanhappy8520** ![][flag_TW] | **4** | 0 | ![][flag_MY] \[Zeth\] | -win by default- |
+| **Maitgon** ![][flag_ES] | **4** | 0 | ![][flag_TW] seanhappy8520 | [#1](https://osu.ppy.sh/community/matches/55339690) |
+| Beat43210 ![][flag_AU] | 0 | 0 | ![][flag_MY] \[Zeth\] | -nullified- |
+| **cheese salad** ![][flag_US] | **4** | 0 | ![][flag_BR] Kyoumo | -win by default- |
+| **Foxeru** ![][flag_BR] | **4** | 0 | ![][flag_FR] maximaxiU | [#1](https://osu.ppy.sh/community/matches/55347203) |
+| **AlvaroYL** ![][flag_ES] | **4** | 1 | ![][flag_BR] Skull Kid | [#1](https://osu.ppy.sh/community/matches/55347205) |
+| **Skull Kid** ![][flag_BR] | **4** | 0 | ![][flag_FR] maximaxiU | [#1](https://osu.ppy.sh/community/matches/55351516) |
+
+| Sunday, 2019-10-06 |  |  |  |  |
+| --: | :-: | :-: | :-- | :-- |
+| Kyoumo ![][flag_BR] | 2 | **4** | ![][flag_JP] **Saikoro** | [#1](https://osu.ppy.sh/community/matches/55358888) |
+| **cheese salad** ![][flag_US] | **4** | 3 | ![][flag_CA] janitoreihil | [#1](https://osu.ppy.sh/community/matches/55359856) |
+| AlvaroYL ![][flag_ES] | 2 | **4** | ![][flag_BR] **Foxeru** | [#1](https://osu.ppy.sh/community/matches/55366733) |
+| **seanhappy8520** ![][flag_TW] | **4** | 1 | ![][flag_JP] Saikoro | [#1](https://osu.ppy.sh/community/matches/55371169) |
+| **POCARI-SWEAT** ![][flag_KR] | **4** | 0 | ![][flag_BR] Skull Kid | [#1](https://osu.ppy.sh/community/matches/55371430) |
+| **janitoreihil** ![][flag_CA] | **4** | 0 | ![][flag_MY] \[Zeth\] | -win by default- |
+| **AlvaroYL** ![][flag_ES] | **4** | 0 | ![][flag_GB] Metzo | [#1](https://osu.ppy.sh/community/matches/55376101) |
+
+---
+
+## Ruleset
```

Comment:
there are always some grammar problems in rulesets, but I'm not sure I should bother to edit them since they are what the tournament used...